client/src/components/atoms/TagInput.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent, useState } from "react";
+import DeletableTag from "./DeletableTag";
+import { TextFieldWithAction } from "./TextFieldWithAction";
+
+interface TagInputProps {
+    tagStrings: string[],
+    onChange: (value: string) => boolean,
+    onSubmit: (value: string) => void,
+    onDelete: (id: number) => void,
+    validateInput: boolean
```

Comment:
Could we change the rename the prop `validateInput` to `isErroneous` since it's a boolean and not a function?